soql-standalone-parser/src/main/scala/com/socrata/soql/parsing/RecursiveDescentHintParser.scala

```diff
@@ -26,8 +26,8 @@ trait RecursiveDescentHintParser { this: RecursiveDescentParser =>
     reader.first match {
       case x@RecursiveDescentParser.MATERIALIZED() =>
         ParseResult(reader.rest, (Materialized(x.position), x.position))
-      case x@RecursiveDescentParser.UNIQUE_ORDER() =>
-        ParseResult(reader.rest, (UniqueOrder(x.position), x.position))
+      case x@RecursiveDescentParser.NO_ROLLUP() =>
```

Comment:
does this section need a case for the NoChainMerge hint?

Reply:
yes, it does

Reply:
🙈 I feel like this is the first time in my life I've said something useful on a code review for you